core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -125,18 +125,34 @@ class LoadBalancerService(config: WhiskConfig, instance: InstanceId, entityStore
    */
   private def processCompletion(response: Either[ActivationId, WhiskActivation],
                                 tid: TransactionId,
-                                forced: Boolean): Unit = {
+                                forced: Boolean,
+                                invoker: InstanceId): Unit = {
     val aid = response.fold(l => l, r => r.activationId)
+
+    // treat left as success (as it is the result a the message exceeding the bus limit)
+    // treat timed out active ack as failure to let the invoker become unhealthy
+    val isSuccess = response.fold(l => true, r => !r.response.isWhiskError)
```

Comment:
timed out active ack === forced.
but this fold does not account for the forced value.
comment inconsistent.